### PR TITLE
Add support for multiple devices by selecting by USB bus and device address

### PR DESCRIPTION
**Pull request by wsneijers**
I want to use the tool on a system with multiple FTDI devices (all HS3) connected. But right now it's not possible because it only can select the first FTDI device. Adding support to select by bus and device address allows me this functionality.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -89,11 +93,21 @@ int main(int argc, char *argv[]) {
...
       break;
     case 'V':
+      use_bus_addr = false;
       sscanf(optarg, "%x", &ftdi_vid);
       break;
     case 'p':
+      use_bus_addr = false;
       sscanf(optarg, "%x", &ftdi_pid);
       break;
+    case 'B':
```
Comment (jiegec) [inline src/main.cpp:106]: Maybe we can emit some error/warning messages if the user specified both VID+PID and dev+bus?

File: src/main.cpp [hunk 2]
```diff
@@ -110,6 +124,8 @@ int main(int argc, char *argv[]) {
       fprintf(stderr, "\t-c A|B|C|D: Select ftdi channel\n");
       fprintf(stderr, "\t-V VID: Specify usb vid\n");
       fprintf(stderr, "\t-p PID: Specify usb pid\n");
+      fprintf(stderr, "\t-B BUS: Specify usb bus addr\n");
+      fprintf(stderr, "\t-D DEV: Specify usb device addr\n");
```
Comment (jiegec) [inline src/main.cpp:133]: Thanks, we will need some documentation for this in README.

File: src/main.cpp (hunk 1 above)
Comment (wsneijers) [reply src/main.cpp:106]: Agreed. Added a error with exit 1 when this happens.

File: src/main.cpp (hunk 2 above)
Comment (wsneijers) [reply src/main.cpp:133]: Added a line in the documentation.

**Pull request merged**